PatchSelectWidget.js

```diff
@@ -0,0 +1,161 @@
+/* global OO, $ */
+
+window.PatchSelectWidget = function PatchSelectWidget( config ) {
+	var widget = this;
+
+	// Parent constructor
+	window.PatchSelectWidget.super.call( this, $.extend( {
+		allowArbitrary: true,
+		inputPosition: 'outline',
```

Comment:
I prefer the look of inline to outline, but also I want each task to be on a separate line so they don't jump when the names arrive from the api.

Reply:
I've pushed an inline version on top of this, maybe the jumping is ok?